src/exposition/http.rs

```diff
@@ -248,6 +270,75 @@ mod handlers {
         Ok(content)
     }
 
+    pub async fn binary_metadata() -> Result<impl warp::Reply, Infallible> {
+        let mut metadata = Metadata::new();
+
+        for metric in &metriken::metrics() {
+            let any = match metric.as_any() {
```

Comment:
Not necessarily for this patch, but I think the checks for inclusion should be centralized since if we accidentally go out of sync between metadata and data, it will be bad,

Reply:
Agree. Added a TODO for this.